client/src/pages/Home.js

```diff
@@ -75,6 +76,13 @@ export function Home() {
 		setIsUploadFailure(true);
 	};
 
+	const showOnClientInfoClick = (i) => {
+		if (isHiden == i) {
```

Comment:
```suggestion
		if (isHiden === i) {
```
Many teams prefer to use only triple equals because double equals can sometimes give surprising results

Reply:
OK, let's remove the surprises.